### DISCUSSION TITLE
How to enable screen to sleep?

### DISCUSSION THREAD

**Discussion by llubi23** [guysoft/FullPageOS#620] [Q&A] [answered]
I’m using Fullpage OS to control my campervan electronics, linked to a website provided by the manufacturer of the system (Victron) it’s perfect every minute of the day. But at night, as it’s programmed to be a kiosk, it’s too bright, so I tried all the commands I found, but none of them works. 
is there any way to make the screen sleep when I want? Command line, button, programmed at X seconds… any way to do it?

thanks!

**Comment by guysoft** [comment, marked as answer]
You can re-enable power management for a start (requires a restart):
https://github.com/guysoft/CustomPiOS/blob/devel/src/modules/gui/filesystem/opt/scripts/start_gui#L4
But that would put the screen dark during the day too. so I guess you will need some way to turn this on an off depending on time of day. Not sure how to do that without some scripting.

**Reply by apartmedia** [reply]
> https://github.com/guysoft/CustomPiOS/blob/devel/src/modules/gui/filesystem/opt/scripts/start_gui

btw (a bit ot)… is this a typo?:
https://github.com/guysoft/CustomPiOS/blob/528d29ea45d2e68ce7e419374697c2ace12c4f74/src/modules/gui/filesystem/opt/scripts/start_gui#L23

**Reply by guysoft** [reply]
Yes seems so

**Reply by guysoft** [reply]
Looks like its consistent at least:
```bash
$  modules git:(devel) grep GUI_SESSION_PLACEHOLDEFR * -r
admin-toolkit/start_chroot_script:    if grep -q "GUI_SESSION_PLACEHOLDEFR" /home/pi/scripts/start_gui ; then
admin-toolkit/start_chroot_script:        sed -i /home/pi/scripts/start_gui -e "s/^GUI_SESSION_PLACEHOLDEFR/export DISPLAY=:0/"
admin-toolkit/start_chroot_script:        echo 'GUI_SESSION_PLACEHOLDEFR' >> /home/pi/scripts/start_gui
gui/filesystem/opt/scripts/start_gui:GUI_SESSION_PLACEHOLDEFR
gui/end_chroot_script:sed -i "s@GUI_SESSION_PLACEHOLDEFR@${GUI_STARTUP_SCRIPT}@g" /opt/custompios/scripts/start_gui
$  modules git:(devel) grep GUI_SESSION_PLACEHOLDER * -r
$  modules git:(devel)
```

**Reply by llubi23** [reply]
Thanks! It works perfect. Not a problem if it blanks during day, I can change minutes and just with a push it’s ON again. 

Thanks again!

**Reply by guysoft** [reply]
Great, also fixed the typo @apartmedia spotted
